adoc/deployment-bare-metal.adoc

```diff
@@ -0,0 +1,67 @@
+include::entities.adoc[]
+:isofile: SLE-15-SP1-Installer-DVD-x86_64-Build225.1-Media1.iso
+:isolink: http://download.suse.de/install/SLE-15-SP1-Installer-GMC1/
+
+== Deployment on VMWare
+
+=== Environment description
+
+[IMPORTANT]
+====
+These instructions currently do not describe how to set up a load balancer.
+This will be added in future versions. You must provide your own load balancing
+solution that directs access to the master nodes.
+====
+
+[NOTE]
+====
+The autoyast file generated by skuba is a template. It has the base requirements. 
```

Comment:
Is the skuba already capable of generating this template?

Reply:
no. wording updated.